src/index.ts

```diff
@@ -0,0 +1,64 @@
+import * as  amqp from 'amqplib/callback_api';
+import { Channel, Connection, Message } from 'amqplib/callback_api';
+import { config } from 'dotenv'; config(); //initialize right there the configuration
+import { QueueMessage } from './types';
+import { PayloadSchema } from './schemas';
+import { connectToDatabase } from './datasource';
+import { Logger } from './utils';
+import * as Joi from 'joi';
+const utils = require('utils')._;
```

Comment:
what's this doing?

Reply:
`const payloadFileContent = utils.tryRead(payloadFile);`

That's where the utils module is being used. It uses the old type of node exports hence cannot be used with the current Typescript way of importing. The logic is encapsulated inside the _ property of the module.

Reply:
It's a safer/shorter way of handling file reading errors.

Reply:
:ok: